src/string_func.h

```diff
@@ -85,7 +85,7 @@ static inline size_t ttd_strnlen(const char *str, size_t maxlen)
 	return t - str;
 }
 
-char *md5sumToString(char *buf, const char *last, const uint8 md5sum[16]);
+std::string Md5sumToString(const uint8 md5sum[16]);
```

Comment:
```suggestion
std::string MD5sumToString(const uint8 md5sum[16]);
```

lower case 'd' looks odd